### PR TITLE
KIALI-2321 Better support smaller screen widths

### PR DESCRIPTION
**Pull request by jshaughn**
@jmazzitelli , for now just let me know if this layout prevents your graph toolbar from wrapping in your Jira environment.  Make sure to test with longest menu values.

### CODE REVIEW

**Comment by jmazzitelli** [comment]
been 5 days, 2 days since I approved - what's the hold up?

**Comment by abonas** [comment]
if this is blocking other things, then you can merge and then add additional improvements if needed after UX discussion. But certainly, we should consult UX on this.

File: src/components/GraphFilter/GraphFilter.tsx [hunk 1]
```diff
@@ -131,15 +135,19 @@ export class GraphFilter extends React.PureComponent<GraphFilterProps> {
...
-          {this.props.node && (
-            <FormGroup className={zeroPaddingLeft}>
-              <Button className={namespaceStyle} onClick={this.handleNamespaceReturn}>
-                Back to Full Graph...
-              </Button>
-            </FormGroup>
-          )}
-          <FormGroup className={zeroPaddingLeft}>
-            <GraphSettingsContainer edgeLabelMode={this.props.edgeLabelMode} graphType={this.props.graphType} />
+          <FormGroup className={breadcrumbStyle} style={{ ...thinGroupStyle }}>
```
Comment (mtho11) [inline src/components/GraphFilter/GraphFilter.tsx]: Using both the **className** and the **style** is probably not the best way to do this.

File: src/components/GraphFilter/GraphFind.tsx [hunk 2]
```diff
@@ -36,6 +36,16 @@ type ParsedExpression = {
   selector: string;
 };
 
+const inputWidth = {
```
Comment (mtho11) [inline src/components/GraphFilter/GraphFind.tsx]: These should be **TypeStyle** functions.

File: src/components/GraphFilter/GraphSettings.tsx [hunk 3]
```diff
@@ -175,16 +175,17 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps> {
       </Popover>
     );
 
-    const alignWithGraphHeaderStyle = style({ marginLeft: -40, paddingRight: '0.8em' });
+    // const alignWithGraphHeaderStyle = style({ marginLeft: -60, paddingRight: '0.5em' });
```
Comment (mtho11) [inline src/components/GraphFilter/GraphSettings.tsx]: Probably didn't mean to leave the commented code.

File: src/components/GraphFilter/GraphSettings.tsx [hunk 4]
```diff
@@ -175,16 +175,17 @@ class GraphSettings extends React.PureComponent<GraphSettingsProps> {
...
 
     return (
-      <span className={alignWithGraphHeaderStyle}>
-        <OverlayTrigger overlay={graphSettingsPopover} placement="bottom" trigger={['click']} rootClose={true}>
-          <Button id="graph_settings">
-            Display <Icon name="angle-down" />
-          </Button>
-        </OverlayTrigger>
-      </span>
+      // <span className={alignWithGraphHeaderStyle}>
```
Comment (mtho11) [inline src/components/GraphFilter/GraphSettings.tsx]: More unused code.

File: src/components/GraphFilter/GraphFilter.tsx (hunk 1 above)
Comment (jshaughn) [reply src/components/GraphFilter/GraphFilter.tsx]: So, I'm not too savvy with this stuff.  But when I tried to combine it into one className style, or all style properties, it just didn't seem to do what I want.  So in general I'm not using styles, but I probably should.  Instead I'm using this "style" objects and a spread operator.  I'd be happy to improve this but I may need help, or a PR to this PR.

**Comment by jshaughn** [comment]
@mattmahoneyrh I think this PR is ready for QE acceptance.  Here is a before/after shot of the graph toolbar, and a list of changes to look for:

Before:
image
After:
image

- Applied to Graph page only:
  - Do not wrap toolbar on 1280x1084 resolution (mazz can verify this)
    - if true for graph it will automatically be true for other main pages as the have more available space
  - Remove the clear icon (the X) for find/hide when the input is empty.
  - Reduce the Find/Hide input box widths
  - Swap the graph type and display buttons for a more logical ordering.
  - Update the Tour 
    - place the arrow closer to the widgets
    - add a tour stop at Find/Hide
- Applied to all of the main pages:
  - Remove margin on left and right.  In other words, left align widget #1 with breadcrumb.  Right align refresh button with toollbar edge.
  - Reduce padding between input groups (the space around the vertical bars) from 20px to 10px
- Applied to all pages using duration/refresh dropdowns
  - Remove the "Display" title for Duration/Refresh dropdowns and add tooltips.
  - Change "min" and "sec" to more standard duration notations in the dropdowns.
    - For example, "Last 1m", "Last 1h", Last "1d", "Every 15s", "Every 5m".

**Pull request merged**